docs/specs/tripgo.swagger.yaml

```diff
@@ -1768,9 +1770,23 @@ definitions:
       openingHours:
         $ref: '#/definitions/OpeningHours'
       pricingTable:
-        $ref: '#/definitions/PricingTable'
+        type: array
+        items:
+          $ref: '#/definitions/PricingTable'
+      restrictions:
+        description: List of car park restrictions.
+        type: array
+        items:
+          type: string
+      entrances:
+        type: array
+        items:
+          $ref: '#/definitions/Entrance'
       source:
         $ref: '#/definitions/DataSourceAttribution'
+      info:
+        type: string
+        description: Additional information about this car park.
     required:
```

Comment:
Are any of the new fields required or all they all optional?

Reply:
@nighthawk added `address` as a required field.